### DISCUSSION TITLE
How to update Scrypted Home Assistant Add-on?

### DISCUSSION THREAD

**Discussion by GizmoGoody** [koush/scrypted#1464] [Q&A] [answered]
How do I update from v99 to v102 in the Home Assistant Scrypted Add-on? I don't see an actual update button. Clicking More Information goes to an error page indicating "github.com refused to connect"

<img width="455" alt="image" src="https://github.com/koush/scrypted/assets/46235745/e35f1019-4645-4394-8097-fb4c338b10aa">

**Comment by skynet01** [comment]
You have to update the addon through home assistant since that's where it's managed from. You have to contact the author of the HA addon and ask them when they will update their addon to the new version.

**Reply by fbnksr** [reply]
The author is @koush as the add-on is being installed via this Github repo. I am also curious as to how to kick off the update within the Home Assistant Add-on version.

**Reply by fbnksr** [reply, marked as answer]
Update got pushed through:
<img width="1552" alt="Screenshot 2024-05-07 at 18 40 06" src="https://github.com/koush/scrypted/assets/25393668/311acb9a-6ac2-4142-847e-c77dae28bbed">
